ses.tf

```
@@ -13,29 +13,13 @@ resource "aws_ses_domain_dkim" "main" {
 resource "cloudflare_record" "devops_care_dkim" {
```

to follow the notation

```suggestion
resource "cloudflare_record" "ses_dkim_devops_care" {
```